dubbo-registry/dubbo-registry-multicast/src/test/java/org/apache/dubbo/registry/multicast/MulticastRegistryTest.java

```diff
@@ -220,4 +221,40 @@ public void testCustomedPort() {
         }
     }
 
+    @Test
+    public void testMulticastAddress() {
+        InetAddress multicastAddress = null;
+        MulticastSocket multicastSocket = null;
+        try {
+            // ipv4 multicast address
+            try {
+                multicastAddress = InetAddress.getByName("224.55.66.77");
+                multicastSocket = new MulticastSocket(2345);
+                multicastSocket.setLoopbackMode(false);
+                NetUtils.setInterface(multicastSocket, multicastAddress);
+                multicastSocket.joinGroup(multicastAddress);
+            } finally {
+                if (multicastSocket != null) {
+                    multicastSocket.close();
+                }
+            }
+
+            // multicast ipv6 address,
+            /*try {

```

Comment:
why those code are removed? are we still need `-Djava.net.preferIPv4Stack=true` after this change are made?

Reply:
No, only because this UT can not pass in travis with ipv6 address. I think the travis host does not have an interface supporting ipv6.